gammapy/modeling/models/core.py

```diff
@@ -182,22 +183,43 @@ def __str__(self):
 
     def __add__(self, other):
```

Comment:
I think the `__add__` method could call `self.extend()` and `self.append()` now.

Reply:
Not changed, as we want to return a new object rather than updating the first one